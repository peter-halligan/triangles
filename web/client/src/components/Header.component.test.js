import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import { Header } from "./Header.component";

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Header />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it(' matches the snapshot ',  () =>{
  const tree = renderer
    .create(<Header />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});