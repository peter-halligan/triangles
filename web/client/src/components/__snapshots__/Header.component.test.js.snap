// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` matches the snapshot  1`] = `
<div
  className="header"
>
  <h1>
    Triangle Type Checker
  </h1>
</div>
`;
